agent/config/default.go

```diff
@@ -276,15 +277,3 @@ func DevConsulSource() Source {
 	`,
 	}
 }
-
-func DefaultRuntimeConfig(hcl string) *RuntimeConfig {
```

Comment:
This was only used by tests, so it was moved to be an unexported test helper next to the two files that used it. The implementation was changed to use `Load()`.